config/routes.rb

```diff
@@ -8,6 +8,7 @@
   namespace :api do
     namespace :v1 do
       resources :podcasts, only: [:index, :create, :show]
+      resources :reviews, only: [:create, :new]
```

Comment:
You don't actually need `:new` here, please remove it!

Reply:
done!